tests/e2e/helpers/decompressors/brotli-database-populator.ts

```diff
@@ -0,0 +1,122 @@
+import * as proto from 'protobufjs';
+import { pack as msgpackrPack } from 'msgpackr';
+import * as brotli from 'brotli-unicode';
+import * as fflate from 'fflate';
+import * as fs from 'fs';
+import { BaseDatabasePopulator } from './base-decompressors-populator';
+
+const COMPRESSED_PREFIX = 'Comp';
+const BROTLI_PREFIX = 'BROTLI';
+
+export class BrotliDatabasePopulator extends BaseDatabasePopulator {
+
+    protected async createCompressedKeys(): Promise<void> {
```

Comment:
I think that it is better to rename this method according to compressor
cause you have the same name createCompressedKeys() for all compressors

Reply:
the name is from baseClass it is an abstract method that must be override in children class